Runtime/Core/Shared/ProcessManagement/ProcessWrapper.cs

```diff
@@ -26,9 +26,7 @@ public string GetProcessOutput(ProcessStartInfo startInfo)
         {
             try
             {
-                return Process.Start(startInfo)
-                .StandardError
-                .ReadLine();
+                return Process.Start(startInfo)?.StandardError?.ReadToEnd() ?? "";
```

Comment:
This might introduce some weird invariants. Return a nullable string instead: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/nullable-value-types

Reply:
Sure. Update method and interface to return `string?`